cmd/mimir/config-descriptor.json

```diff
@@ -3063,6 +3052,17 @@
           "fieldType": "int",
           "fieldCategory": "experimental"
         },
+        {
+          "kind": "field",
+          "name": "accept_native_histograms",
+          "required": false,
+          "desc": "Flag to enable, per-tenant, the ingestion of native histogram samples.",
```

Comment:
Whether or not this is per-tenant depends on which limits config is being used (runtime vs the global config). Best to just not mention it.